### PR TITLE
add Mainapp class

### PR DESCRIPTION
**Pull request by sofia-shchukina**

### CODE REVIEW

**Review by davidebschke — COMMENTED** [review]
New Comment from me

**Review by davidebschke — APPROVED** [review]
All Good

File: src/main/java/com/example/projectforfuturemerging/MainApp.java [hunk 1]
```diff
@@ -1,6 +1,16 @@
...
+import org.springframework.web.bind.annotation.RestController;
 
+@RestController
+@RequestMapping("/api")
+public class MainApp {
 
+    @GetMapping("lorem")
+    public String lorem() {
+        return "lorem ipsum";
+    }
```
Comment (KSVoss) [inline src/main/java/com/example/projectforfuturemerging/MainApp.java:14]: nice work

**Review by KSVoss — COMMENTED** [review]
is ok

**Pull request merged**